### PR TITLE
M2 conclusions

### PR DESCRIPTION
**Pull request by DoraNicole**

### CODE REVIEW

File: _includes/m2/conclusions.html [hunk 1]
```diff
@@ -9,7 +9,10 @@ <h2 class="section-heading text-uppercase">{{ site.data.m2.conclusions.title }}<
 
       <div class="row">
         <div class="col col-lg-12 text-center">
```
Comment (GeorgeMD) [inline _includes/m2/conclusions.html:11]: @SanduIrina said this should be justify, and I'm on justify team.

**Review by GeorgeMD — CHANGES_REQUESTED** [review]

File: _includes/m2/conclusions.html (hunk 1 above)
Comment (GeorgeMD) [inline _includes/m2/conclusions.html:11]:
```suggestion
        <div class="col col-lg-12 text-justify">
```

**Review by GeorgeMD — CHANGES_REQUESTED** [review]

File: _includes/m2/conclusions.html (hunk 1 above)
Comment (SanduIrina) [inline _includes/m2/conclusions.html]:
```suggestion
        <div class="col col-lg-12 text-justify">
```

**Review by GeorgeMD — APPROVED** [review]

File: _data/m2.yml [hunk 2]
```diff
@@ -43,4 +43,8 @@ solution_validation:
...
+    pr_val: The problems people often have when trying to study is that there is a lot of information on the internet and they do not know where to start. The information is not structured according to their needs, it is sometimes incomplete or taken out of the context. These facts confirm the problem discovered by us. 
+    sol_val: Moreover, when studying something new, most people do not use a certain tool they use to find materials, they have particular methods to find what they need, but they use organizational tools, to structure their tasks. Also, they do not use many extensions, but when they find a very useful one, they stick to that.  The majority of the persons we interviewed would find very useful and time-saving, a tool to see suggestions about what to learn based on their needs, level of knowledge and the reviews from other users.
```
Comment (SanduIrina) [inline _data/m2.yml]:
```suggestion
    sol_val: Moreover, when studying something new, most people do not use a certain tool to find materials, they have particular methods to find what they need, but they use organizational tools, to structure their tasks. Also, they do not use many extensions, but when they find a very useful one, they stick to that. The majority of the persons we interviewed would find a tool to see suggestions about what to learn based on their needs, level of knowledge and the reviews from other users to be very useful and time-saving.
```

**Review by SanduIrina — APPROVED** [review]

**Review by GeorgeMD — APPROVED** [review]

**Pull request merged**